components/Onboarding/Sections/Usability.js

```diff
@@ -0,0 +1,88 @@
+import React, { Fragment } from 'react'
+import { Query, Mutation } from 'react-apollo'
+import { css } from 'glamor'
+
+import { Interaction, mediaQueries, Button } from '@project-r/styleguide'
+
+import Section from '../Section'
+import ProgressSettings from '../../Account/ProgressSettings'
+import {
+  progressConsentQuery,
+  submitConsentMutation,
+  revokeConsentMutation
+} from '../../Article/Progress/api'
+
+const { P } = Interaction
+
+const styles = {
+  p: css({
+    marginBottom: 20
+  }),
+  actions: css({
+    marginBottom: 20,
+    display: 'flex',
+    flexWrap: 'wrap',
+    position: 'relative',
+    '& > button': {
+      flexGrow: 1,
+      margin: '5px 15px 0 0',
+      minWidth: '120px',
+      [mediaQueries.mUp]: {
+        flexGrow: 0,
+        margin: '5px 15px 0 0',
+        minWidth: '160px'
+      }
+    }
+  })
+}
+
+const Usability = (props) => {
+  const { onContinue } = props
+
+  return (
+    <Query query={progressConsentQuery} fetchPolicy={'no-cache'}>
+      {({ loading, error, data = {} }) => {
+        // Is ticked when either REVOKE or GRANT consent was submitted.
+        const hasConsented = data.me && data.me.hasConsentedTo !== null
+
+        return (
+          <Section
+            heading='Lesekomfort'
+            isLoading={loading}
+            error={error}
+            isTicked={hasConsented}
+            showContinue={hasConsented}
+            {...props}>
+            {hasConsented ? (
+              <Fragment>
+                <ProgressSettings />
+                <br />
+              </Fragment>
+            ) : (
+              <Fragment>
+                <P {...styles.p}>Auf Wunsch merken wir uns Ihre Leseeposition in Beiträgen.</P>
+                <P {...styles.p}>So können Sie auf unterschiedlichen Geräten an der letzten Stelle weiterlesen.</P>
+                <P {...styles.p}>Mehr Lesekomfort, für den wir allerdings Ihr persönliches Leseverhalten in der Republik aufzeichnen müssen. Mehr Informationen</P>
+                <div {...styles.actions}>
+                  <Mutation mutation={submitConsentMutation} onCompleted={onContinue.bind(null, [props])}>
+                    {(mutate, { loading }) => {
+                      return <Button onClick={mutate} disabled={loading}>Ja</Button>
+                    }}
+                  </Mutation>
+                  <Mutation mutation={revokeConsentMutation} onCompleted={onContinue.bind(null, [props])}>
+                    {(mutate, { loading }) => {
+                      return <Button onClick={mutate} disabled={loading}>Nein</Button>
+                    }}
+                  </Mutation>
+                </div>
+              </Fragment>
+            )}
+            <P {...styles.p}>Jeden Beitrag können Sie auch auf Ihre Lesezeichen-Liste packen.</P>
```

Comment:
Eher weg lassen hier. Sonst verwirrt es vielleicht. Ist ja in der Gebrauchsanweisungen.